### PR TITLE
Parainfo

### LINKED ISSUE #1256: [STALE] build-build-with-polkadot.md
Last edit happened 21 days ago 😬

### LINKED ISSUE #1304: [STALE] build-parachains-rococo.md
Last edit happened 54 days ago 😬

### LINKED ISSUE #1336: [STALE] learn-auction.md
Last edit happened 45 days ago 😬

### LINKED ISSUE #1510: [STALE] learn-parachains.md
Last edit happened 21 days ago 😬

### LINKED ISSUE #1697: [STALE] docs/build-build-with-polkadot.md
Last edit happened 87 days ago 😬

### LINKED ISSUE #1821: [STALE] docs/learn-parachains.md
Last edit happened 50 days ago 😬

### LINKED ISSUE #1838: [STALE] docs/learn-crowdloans.md
Last edit happened 45 days ago 😬

### LINKED ISSUE #1900: Add parachain auction and crowdloans info to Kusama
Specifically https://wiki.polkadot.network/docs/en/learn-auction and https://wiki.polkadot.network/docs/en/learn-crowdloans, changing the parameters for Kusama of course

### PR DESCRIPTION
**Pull request by Swader**
- [x] Check that the conditional rendering worked on merge
- [x] Double check that there's no 404s 

Closes #1336 
Closes #1510
Closes #1821
Closes #1304
Closes #1838
Closes #1900
Closes #1697
Closes #1256 

- [ ] Double check before merge that the lease for Kusama is being extended to 3 months, and make changes where needed.

### CODE REVIEW

**Comment by jakehemmerle** [comment]
build-parachains-rococo.md under "Lateral Transfers" : 'page here' link is broken. Not sure the context of it but it doesn't seem totally relevant and can probably be removed

**Comment by Swader** [comment]
Leases on Kusama might extend to 3 months instead. Needs re-check before merge.

File: docs/build-build-with-polkadot.md [hunk 1]
```diff
@@ -354,7 +356,8 @@ the [Polkadot Watercooler](https://riot.im/app/#/room/#polkadot-watercooler:matr
 about them. Developers may be interested in joining the
 [Polkadot Beginners Lounge](https://riot.im/app/#/room/#polkadotnoobs:matrix.org) or
 [Substrate Technical](https://riot.im/app/#/room/#substrate-technical:matrix.org) to ask their
-questions. As always, keep up to date with Polkadot by following the [social channels](community).
+questions. As always, keep up to date with Polkadot and Kusamaby following the
```
Comment (laboon) [inline docs/build-build-with-polkadot.md]:
```suggestion
questions. As always, keep up to date with Polkadot and Kusama by following the
```

File: docs/build-parachains-rococo.md [hunk 2]
```diff
@@ -122,80 +112,152 @@ initial tests are successful on Rococo, we envision that in the long run it will
...
-   [Rococo Extrinsics](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frococo-rpc.polkadot.io#/extrinsics)
-1. Follow the
-   [registration process](https://github.com/substrate-developer-hub/cumulus-workshop/blob/master/en/6-register/1-register.md#request-parachain-registration)
-1. You are free to do runtime upgrades after the parachain is connected, so you can still iterate on
-   features later on
+Below you will find a general plan on how we approach onboarding parachains within this new
+environment.
+
+## Relevant Values and Calculations
+
```
Comment (laboon) [inline docs/build-parachains-rococo.md:122]:
```suggestion

Note that these are example values; all are subject to change.

```

File: docs/build-parachains-rococo.md [hunk 3]
```diff
@@ -122,80 +112,152 @@ initial tests are successful on Rococo, we envision that in the long run it will
...
+After submission of this data, it will take 2 sessions (30 min x 2) for the candidate to fully
+onboard to a Parathread.
+
+> Note: All transitions of a Para into different states will take at least 2 sessions (onboarding,
+> offboarding, upgrading, downgrading, etc.)
+
+## Slots
+
+To start, we will only have 5 or 6 slots available on Rococo to ensure that the chain functions
```
Comment (laboon) [inline docs/build-parachains-rococo.md]:
```suggestion
To start, only 5 or 6 slots are planned to be available on Rococo to ensure that the chain functions
```

File: docs/build-parachains-rococo.md [hunk 4]
```diff
@@ -122,80 +112,152 @@ initial tests are successful on Rococo, we envision that in the long run it will
...
+- `period_begin`: The first lease period for the slot.
+- `period_count`: The number of lease periods to reserve.
+
+So assuming we start at genesis, and Trick is `ParaId(0)` something like:
+
+```
+force_lease(ParaId(0), Alice, 0, 0, 100)
+```
+
+Would give Trick 100 lease periods, starting at lease period 0, but it wouldn't start until period 1
```
Comment (laboon) [inline docs/build-parachains-rococo.md]:
```suggestion
would give Trick 100 lease periods, starting at lease period 0, but it wouldn't start until period 1
```

File: docs/learn-auction.md [hunk 5]
```diff
@@ -75,10 +76,11 @@ Polkadot will use a _random beacon_ based on the VRF that's used also in other p
 protocol. The VRF will provide the base of the randomness, which will retroactively determine the
 end-time of the auction.
 
-The slot durations are capped to 2 years and divided into 6-month periods. Parachains may lease a
-slot for any contiguous range of the slot duration. Parachains may lease more than one slot over
-time, meaning that they could extend their lease to Polkadot past the 2 year slot duration simply by
-leasing a contiguous slot.
+The slot durations are capped to {{ polkadot: 2 years and divided into 6-month periods :polkadot }}
+{{ kusama: 24 weeks and divided into 6-week periods :kusama }}. Parachains may lease a slot for any
```
Comment (laboon) [inline docs/learn-auction.md]:
```suggestion
The slot durations are capped to {{ polkadot: 2 years and divided into 6-month periods :polkadot }}; 
Kusama slots durations and periods will be less than on Polkadot. Parachains may lease a slot for any
```

File: docs/learn-auction.md [hunk 6]
```diff
@@ -101,15 +104,17 @@ Slot C |__________|     1    |     2     |     3     |     4     |...
...
+       ---------------------max lease-------------------
 
-Each period of the range 1 - 4 represents a 6-month duration for a total of 2 years
 ```
 
-Each parachain slot has a maximum duration of 2 years, divided into 6-month periods. More than one
-continuous period is a range.
+_Each period of the range 1 - 4 represents a
+{{ polkadot: 6-month duration for a total of 2 years :polkadot }}
+{{ kusama: 6-week duration for a total of 24 weeks :kusama }}_
```
Comment (laboon) [inline docs/learn-auction.md]:
```suggestion
_Each period of the range 1 - 4 represents a
{{ polkadot: 6-month duration for a total of 2 years :polkadot }}
or a shorter duration on Kusama_
```

File: docs/learn-parachains.md [hunk 7]
```diff
@@ -5,88 +5,105 @@ sidebar_label: Parachains
...
 as a Wasm executable. Proofs of new state transitions that occur on a parachain must be validated
 against the registered state transition function (STF) that is stored on the Relay Chain by the
-validators before Polkadot acknowledges a state transition has occurred on a parachain. The only
-constraint to the logic that a parachain is allowed to implement is that it must be verifiable by
-the Relay Chain validators. Verification most commonly takes the form of a bundled proof of a state
-transition known as a Proof-of-Verification (PoV) block, which is submitted to the validators from
-one or more of the parachain collators to be checked.
+validators before {{ polkadot: Polkadot :polkadot }} {{ kusama: Kusama :kusama }} acknowledges a
+state transition has occurred on a parachain. The only constraint to the logic that a parachain is
+allowed to implement is that it must be verifiable by the Relay Chain validators. Verification most
```
Comment (laboon) [inline docs/learn-parachains.md]:
```suggestion
state transition has occurred on a parachain. The key constraint regarding the logic of a parachain is
that it must be verifiable by the Relay Chain validators. Verification most
```
I believe there are other constraints here, e.g. in terms of relying on relay chain finalization,  block production time, etc.

File: docs/learn-parachains.md [hunk 8]
```diff
@@ -5,88 +5,105 @@ sidebar_label: Parachains
...
 
 ## Parachain Economies
 
 Parachains may have their own economies with their own native tokens. Schemes such as Proof-of-Stake
 are usually used to select the validator set in order to handle validation and finalization;
-parachains will not be required to do either of those things. However, since Polkadot is general
-over what the parachain can implement, it may be the choice of the parachain to implement a staking
-token, but it's not generally necessary.
+parachains will not be required to do either of those things. However, since
+{{ polkadot: Polkadot :polkadot }} {{ kusama: Kusama :kusama }} is general over what the parachain
```
Comment (laboon) [inline docs/learn-parachains.md]:
```suggestion
{{ polkadot: Polkadot :polkadot }} {{ kusama: Kusama :kusama }} is not overly particular about what the parachain
```

**Review by laboon — CHANGES_REQUESTED** [review]
Bruno - if you are OK with the changes I made, can you merge when ready?

**Pull request merged**